### PR TITLE
Control Zero F7 - Fix for voltage/current sensing

### PR DESCRIPTION
**Pull request by ryanjAA**
Fix to enable standard current/voltage sensor to display voltage and current readings on Control Zero F7.

### CODE REVIEW

**Review by ryanjAA — COMMENTED** [review]
Removed .

File: boards/mro/ctrl-zero-f7/src/board_config.h [hunk 1]
```diff
@@ -175,7 +175,7 @@
 #define BOARD_ADC_USB_CONNECTED	       (px4_arch_gpioread(GPIO_OTGFS_VBUS))
 #define BOARD_ADC_USB_VALID            BOARD_ADC_USB_CONNECTED
 #define BOARD_ADC_SERVO_VALID          (1)	/* never powers off the Servo rail */
-#define BOARD_ADC_BRICK_VALID          (!px4_arch_gpioread(GPIO_nVDD_BRICK1_VALID))
+#define BOARD_ADC_BRICK_VALID          (px4_arch_gpioread(GPIO_nVDD_BRICK1_VALID))
```
Comment (dagar) [inline boards/mro/ctrl-zero-f7/src/board_config.h:178]: So `GPIO_nVDD_BRICK1_VALID` is actually `GPIO_VDD_BRICK1_VALID`?

Comment (ryanjAA) [reply boards/mro/ctrl-zero-f7/src/board_config.h]: Ok, it is renamed to GPIO_VDD_BRICK1_VALID (if I understood correctly that is what you were after but that would be different than the naming convention of everything else I could see).

**Comment by dagar** [comment]
Confirmed working?

 - `listener system_power`
 - `listener battery_status`

**Comment by ryanjAA** [comment]
Confirmed working.

Screen Shot 2021-01-19 at 2 31 29 PM
Screen Shot 2021-01-19 at 2 30 00 PM

File: boards/mro/ctrl-zero-f7/src/board_config.h [hunk 2]
```diff
@@ -112,7 +112,7 @@
 /* Power supply control and monitoring GPIOs */
 #define GPIO_nPOWER_IN_A                /* PB5 */ (GPIO_INPUT|GPIO_PULLUP|GPIO_PORTB|GPIO_PIN5)
 
-#define GPIO_nVDD_BRICK1_VALID          GPIO_nPOWER_IN_A /* Brick 1 Is Chosen */
+#define GPIO_VDD_BRICK1_VALID          GPIO_nPOWER_IN_A /* Brick 1 Is Chosen */
```
Comment (dagar) [inline boards/mro/ctrl-zero-f7/src/board_config.h:115]:
```suggestion
#define GPIO_VDD_BRICK1_VALID          /* PB5 */ (GPIO_INPUT|GPIO_PULLUP|GPIO_PORTB|GPIO_PIN5)
```

File: boards/mro/ctrl-zero-f7/src/board_config.h [hunk 3]
```diff
@@ -112,7 +112,7 @@
 /* Power supply control and monitoring GPIOs */
 #define GPIO_nPOWER_IN_A                /* PB5 */ (GPIO_INPUT|GPIO_PULLUP|GPIO_PORTB|GPIO_PIN5)
```
Comment (dagar) [inline boards/mro/ctrl-zero-f7/src/board_config.h:113]:
```suggestion
```

**Review by dagar — APPROVED** [review]

**Pull request merged**